projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -289,6 +290,9 @@ public static function load_verbum_comments_admin() {
 	 * @return void
 	 */
 	public static function load_wpcom_command_palette() {
+		if ( function_exists( 'is_agency_managed_site' ) && is_agency_managed_site() ) {
+			return;
+		}
```

Comment:
Removes the Command Palette.

<img src="https://private-user-images.githubusercontent.com/5634774/341850564-f4e44ccf-6b62-4956-ab98-bb8c4f201510.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkwMDkzOTcsIm5iZiI6MTcxOTAwOTA5NywicGF0aCI6Ii81NjM0Nzc0LzM0MTg1MDU2NC1mNGU0NGNjZi02YjYyLTQ5NTYtYWI5OC1iYjhjNGYyMDE1MTAucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYyMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MjFUMjIzMTM3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NTU1MzYxNDZlNDQ4N2M4MGYzODNiYzQ5MGU0YjI3MmJkMmJjODhjNzEwNmFjYTgzZGJkYTBmMDBiOGIxNjg3NCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.D8WFxZVKR-JdhtTOobcvPnVkFwIXuf_sj0co1YhsWAw" />